service/src/main/java/uk/gov/hmcts/reform/fpl/Application.java

```diff
@@ -10,7 +10,8 @@
 @EnableFeignClients(basePackages = {
     "uk.gov.hmcts.reform.idam.client",
     "uk.gov.hmcts.reform.rd.client",
-    "uk.gov.hmcts.reform.fnp.client"
+    "uk.gov.hmcts.reform.fnp.client",
+    "uk.gov.hmcts.reform.payment.client"
```

Comment:
thoughts on combining this with the `fnp` package? Or should `fnp` be changed to just `fee`?